src/pretty-dom.js

```diff
@@ -0,0 +1,16 @@
+import prettyFormat from 'pretty-format'
+
+const {DOMElement, DOMCollection} = prettyFormat.plugins
+
+function prettyDOM(htmlElement, maxLength = undefined) {
```

Comment:
Don't need the `= undefined` because that's functionally equivalent to what the value of `maxLength` would be anyway :)

Reply:
I know, I just like to be more explicit in the declaration that the argument is not expected, to anyone that passes by and read the code. Guess flow or typescript would be clearer for that purpose but I have yet to learn to use one of them, it's on my list ;)